### DISCUSSION TITLE
https://maxkim-j.github.io/posts/i-definitely-do-sth-wrong

### DISCUSSION THREAD

**Discussion by giscus** [MaxKim-J/maxkim-j.github.io#96] [Announcements]
# https://maxkim-j.github.io/posts/i-definitely-do-sth-wrong

낮은 에고, 자기 혐오, 겸손함

https://maxkim-j.github.io/posts/i-definitely-do-sth-wrong

**Comment by hanameee** [comment]
스스로를 향하는 부정은 도망칠 곳이 없어서 더 힘든것 같아요 허허… 잘 격려받지 못함에도 격려하려는 마음을 갖고 계시니 언젠간 셀프 칭찬감옥에 갇히시리라 믿습니닷

**Reply by hg-pyun** [reply]
따뜻한 해나님...

**Reply by jeff-tbd** [reply]
따뜻한 해나님...

**Reply by MaxKim-J** [reply]
@hanameee 따뜻한 말씀 감사합니당 헤헤 언젠가 스스로를 인정하고 좋게 봐주는 날이 올 수도 있을거라고 생각해요
반면에 오지 않더라도 지금처럼 행복할 수 있으리라 믿기도 하고요~~

**Reply by MaxKim-J** [reply]
ㅋㅋㅋㅋㅋㅋㅋ여기서 다들 뵈니 반갑네요...

**Reply by eschocolat** [reply]
따뜻한 해나님..
